projects/1.21-fabric/assets/archers-expansion/archers_expansion/lang/zh_cn.json

```
@@ -0,0 +1,97 @@
+    "item.archers_expansion.deadeye_legs": "神射手护腿",
+    "item.archers_expansion.deadeye_feet": "神射手靴子",
+    "item.archers_expansion.netherite_deadeye_chest": "下界合金神射手外套",
+    "item.archers_expansion.netherite_deadeye_head": "下界合金神射手外套",
+    "item.archers_expansion.netherite_deadeye_legs": "下界合金神射手外套",
+    "item.archers_expansion.netherite_deadeye_feet": "下界合金神射手外套",
+  
+    "spell.archers_expansion.choking_gas.name": "窒息毒雾",
+    "spell.archers_expansion.choking_gas.description": "在弹射物命中时生成一片毒雾，造成 {damage} 点伤害。",
+    "spell.archers_expansion.disabling_shot.name": "致残射击",
+    "spell.archers_expansion.disabling_shot.description": "命中目标后向后跳跃，使目标减速并造成 {damage} 点伤害。潜行时可取消后跳。",
+    "spell.archers_expansion.fast_shot.name": "快速射击",
+    "spell.archers_expansion.fast_shot.description": "你的接下来几次射击会获得移速和射击速度效果",
+    "spell.archers_expansion.trick_shot.name": "诡跃射击",
+    "spell.archers_expansion.trick_shot.description": "生成一支弹跳箭矢，在目标间弹射并造成 {damage} 点伤害。",
+  
+    "effect.archers_expansion.fast_shot": "快速射击",
+    "effect.archers_expansion.fast_shot.description": "每次射击都会增加射击速度和移速。",
+    "effect.archers_expansion.choking_gas": "窒息毒雾",
+    "effect.archers_expansion.choking_gas.description": "通过中毒对目标造成伤害并使其沉默。处于流血状态时伤害提升。",
+    "effect.archers_expansion.leaping_shot": "后跃射击",
+    "effect.archers_expansion.leaping_shot.description": "施法者向后跳跃，潜行时可取消。",
+    "effect.archers_expansion.disabling_shot": "致残射击",
+    "effect.archers_expansion.disabling_shot.description": "减速目标并对其持续造成伤害",
+  
+    "item.archers_expansion.tundra_hunter_spell_book": "寒霜狩典",
+    "item.archers_expansion.tundra_hunter.spell_scroll": "寒霜猎人卷轴",
+    "item.archers_expansion.tundra_hunter_chest": "寒霜外套",
+    "item.archers_expansion.tundra_hunter_head": "寒霜兜帽",
+    "item.archers_expansion.tundra_hunter_legs": "寒霜护腿",
+    "item.archers_expansion.tundra_hunter_feet": "寒霜靴子",
+    "item.archers_expansion.netherite_tundra_hunter_chest": "下界合金寒霜外套",
+    "item.archers_expansion.netherite_tundra_hunter_head": "下界合金寒霜兜帽",
+    "item.archers_expansion.netherite_tundra_hunter_legs": "下界合金寒霜护腿",
+    "item.archers_expansion.netherite_tundra_hunter_feet": "下界合金寒霜靴子",
+  
+    "spell.archers_expansion.arctic_volley.name": "寒冰箭雨",
+    "spell.archers_expansion.arctic_volley.description": "一次性射出8支寒冰箭，造成 {damage} 点伤害并冻结目标。",
+    "spell.archers_expansion.enchanted_crystal_arrow.name": "附魔水晶箭",
+    "spell.archers_expansion.enchanted_crystal_arrow.description": "造成 {damage} 点伤害，使目标眩晕并冻结。",
+    "spell.archers_expansion.frozen_pact.name": "冰霜契约",
+    "spell.archers_expansion.frozen_pact.description": "根据目标的冰霜层数造成伤害，并在 {effect_duration} 秒内降低其攻击力。",
+    "spell.archers_expansion.frozen_shot.name": "冰霜射击",
+    "spell.archers_expansion.frozen_shot.description": "你的接下来几次射击会为目标叠加冰霜效果，使其减速并逐渐冻结。",
+  
+    "effect.archers_expansion.enchanted_crystal_arrow": "水晶箭矢",
+    "effect.archers_expansion.enchanted_crystal_arrow.description": "定身目标并为其叠加冰霜层数。",

```

```suggestion
    "effect.archers_expansion.enchanted_crystal_arrow.description": "眩晕目标并叠加其冰霜层数。",

```
目前其前置模组的官汉stun用的是眩晕，可以看[这个](https://github.com/ZsoltMolnarrr/SpellEngine/pull/104/files) Line71